php-checks/src/main/java/org/sonar/php/checks/PHPDeprecatedFunctionUsageCheck.java

```diff
@@ -138,4 +143,12 @@ private void checkAssertArguments(FunctionCallTree tree) {
     }
   }
 
+  private void checkDefineArguments(FunctionCallTree tree) {
+    SeparatedList<ExpressionTree> arguments = tree.arguments();
+    if (arguments.size() >= 3 && arguments.get(2).is(Kind.BOOLEAN_LITERAL) &&
```

Comment:
is there a version with more parameter than 3? If not i would use == 3 rather than >=.